doc/man7/EVP_KDF_HKDF.pod

```diff
@@ -15,67 +15,43 @@ and "extracts" from it a fixed-length pseudorandom key K. The second stage
 "expands" the key K into several additional pseudorandom keys (the output
 of the KDF).
 
-=head2 Numeric identity
+=head2 Identity
 
-B<EVP_KDF_HKDF> is the numeric identity for this implementation; it
-can be used with the EVP_KDF_CTX_new_id() function.
+B<SN_hkdf> is the name for this implementation; it
+can be used with the EVP_KDF_new() function.
 
-=head2 Supported controls
+=head2 Supported parameters
 
-The supported controls are:
+The supported parameters are:
 
 =over 4
 
-=item B<EVP_KDF_CTRL_SET_SALT>
+=item B<OSSL_KDF_PARAM_DIGEST>
```

Comment:
I would very much like to see something like this:

```
=item B<OSSL_KDF_PARAM_DIGEST> ("digest") <octet string>
```

Reply:
Very good suggestion.  It will simplify the descriptions.

Reply:
I suggest having a look at how I did with the MAC docs

Reply:
I did look but missed this detail.  I'll change.